source/iostreams.tex

```diff
@@ -5916,6 +5916,19 @@
 \postcondition
 \tcode{rdbuf() == sb}.
 
+\indexlibrary{\idxcode{basic_ostream}!constructor}%
```

Comment:
@AlisdairM: Same question here, how come the synopsis does not list the move constructor?